### PR TITLE
Fix the `null` error in SelectAutoComplete Widget

### PR DESCRIPTION
**Pull request by iFlameing**
@reebalazs I added this optional chaining because what happens props has the choices but its value is null. Then we don't get the array initialization.

### CODE REVIEW

File: src/components/manage/Widgets/SelectAutoComplete.jsx [hunk 1]
```diff
@@ -139,7 +139,7 @@ class SelectAutoComplete extends Component {
     if (
       this.state.termsPairsCache.length === 0 &&
       value?.length > 0 &&
-      choices.length > 0
+      choices?.length > 0
```
Comment (reebalazs) [inline src/components/manage/Widgets/SelectAutoComplete.jsx:142]: ok so if we want to allow null then please remove the default value  = [] as it's not necessary any more.

Also adding a test for the choices=null case would be nice... but might not be trivial, provided that currently there is only a single render test case.

Comment (iFlameing) [reply src/components/manage/Widgets/SelectAutoComplete.jsx:142]: done

**Review by reebalazs — APPROVED** [review]
